### DISCUSSION TITLE
how do i report issues to the creator

### DISCUSSION THREAD

**Discussion by Begodden** [GalaxyOfZero/Skyfriends#25] [General]
being new to minecraft i dont know how to report things to the creator like the Sulfur Recovery Unit in Immersive Engineering the Iron sheet metal it recomends the wrong one it says it neeed the Create Deco but the when building the unit it dont like them pieces

**Comment by GalaxyOfZero** [comment]
i know this Quest Problem, it is Fix in the 2.0 Update, but it takes a lot on time befor i finish the 2.0 Update

**Reply by Begodden** [reply]
ok thanks very hard modpack for a newbie but im enjoying it

**Reply by GalaxyOfZero** [reply]
thanks and enjoy the pack 👍
